nixos/modules/services/networking/minidlna.nix

```diff
@@ -107,6 +109,26 @@ in
     };
   };
 
+  options.services.minidlna.user = mkOption {
+    type = types.str;
+    default = defaultUser;
+    example = "yourUser";
+    description = ''
+      The user to run MiniDLNA as.
+      By default, a user named <literal>${defaultUser}</literal> will be created.
```

Comment:
Perhaps this text could make it clearer that changing the user/group means the user/group won't be created anymore?

Reply:
Great idea; updated.